src/networks.json

```diff
@@ -710,6 +710,21 @@
     "start": 2762929,
     "logo": "ipfs://QmXLZyAr6UNFQ4tkNwSyeNByFvzwYpwiNgV5vHuoxn74Rg"
   },
+  "534": {
+    "key": "534",
+    "name": "Candle",
+    "shortName": "CNDL",
+    "chainId": 534,
+    "network": "mainnet",
+    "multicall": "0x7839444C21C03F03cF7CD29718703D5e9B0B010C",
```

Comment:
Hey, @samisbakedham Looks good, except that the multicall code here is not verified, can you verify and publish the code to explorer plz. :) Thanks

Reply:
Fixed!